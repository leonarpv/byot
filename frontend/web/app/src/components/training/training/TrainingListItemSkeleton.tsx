import * as React from 'react';
import {makeStyles, Theme} from '@material-ui/core';
import {WithStyles} from '@byot-frontend/web-common/src/types/WithStyles';
import {TrainingSetListItemSkeleton} from '../set/TrainingSetListItemSkeleton';

interface Props extends WithStyles<typeof styles> {}

const styles = (theme: Theme) => ({});
const useStyles = makeStyles(styles);

export const TrainingListItemSkeleton: React.FC<Props> = (props: Props) => {
  const styles = useStyles(props);
  return <TrainingSetListItemSkeleton />;
};
