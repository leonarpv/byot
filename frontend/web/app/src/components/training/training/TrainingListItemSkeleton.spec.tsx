import * as React from 'react';
import {render} from '@testing-library/react';
import {TrainingListItemSkeleton} from './TrainingListItemSkeleton';

describe('<TrainingListItemSkeleton/>', () => {
  it('should render', () => {
    const {container} = render(<TrainingListItemSkeleton />);
    expect(container).toMatchSnapshot();
  });
});
