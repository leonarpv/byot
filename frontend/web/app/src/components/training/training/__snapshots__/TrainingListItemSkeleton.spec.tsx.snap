// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrainingListItemSkeleton/> should render 1`] = `
<div>
  <div
    class="MuiBox-root MuiBox-root-5 makeStyles-root-4"
  >
    <div
      class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
    >
      <div
        class="MuiGrid-root MuiGrid-item"
      >
        <span
          class="MuiSkeleton-root makeStyles-root-6 makeStyles-image-1 MuiSkeleton-rect MuiSkeleton-pulse"
        />
      </div>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-1 MuiGrid-item MuiGrid-grid-xs-true"
      >
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <span
            class="MuiSkeleton-root makeStyles-root-6 makeStyles-title-2 MuiSkeleton-rect MuiSkeleton-pulse"
            style="width: 100%;"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <span
            class="MuiSkeleton-root makeStyles-root-6 makeStyles-descriptions-3 MuiSkeleton-rect MuiSkeleton-pulse"
            style="width: 100%;"
          />
        </div>
        <div
          class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
        >
          <span
            class="MuiSkeleton-root makeStyles-root-6 makeStyles-descriptions-3 MuiSkeleton-rect MuiSkeleton-pulse"
            style="width: 100%;"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
